Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj

```diff
@@ -14,7 +14,13 @@
         CSharpier is a net6.0 project with <PackAsTool> and this is neither.
     -->
     <Target Name="BuildCSharpier" BeforeTargets="Build">
-        <Exec command="dotnet build -c $(Configuration)" WorkingDirectory="../CSharpier.Cli" />
+      <!-- 
+          we run these commands in parallel with two frameworks 
+          and letting dotnet build do the restore results in a failure with file locks
```

Comment:
Maybe add a period at the end of this line?
This comment makes a lot more sense when you realize these are 2 separate sentences.

Reply:
I tweaked the comment right on master, it needed a lot more than just a period!